generictry/src/main/java/generictry/Agregate1.java

```diff
@@ -0,0 +1,30 @@
+package generictry;
+
+import generictry.agregate1.A;
+import generictry.agregate1.A2;
+import generictry.agregate1.B;
+import generictry.agregate1.B2;
+
+/**
+ * @author Jean-Aymeric Diet
+ *
+ */
+public class Agregate1 {
+	/**
+	 *
+	 */
+	public void start() {

```

Comment:
\[Checkstyle\] ERROR: Method 'start' is not designed for extension \- needs to be abstract, final or empty\.